src/debian/8/aspnet-benchmarks/mongo/Dockerfile

```diff
@@ -0,0 +1,4 @@
+FROM mongo:3.4.10
+
+ADD create.js /docker-entrypoint-initdb.d/
```

Comment:
`COPY` is preferred over `ADD`.
But this file doesn't even exist, resulting in a build error. So what is the intent here? Same question applies to a bunch of the other Dockerfiles that are using `ADD`.